docs/static/azure-module.asciidoc

```diff
@@ -50,128 +52,97 @@ These {kib} dashboards are available and ready for you to use. You can use them
-The command line configuration uses the configuration options you provide and supports only one Event Hub.
+Specify <<azure_config_options,options>> for the Logstash Azure module in the
+`logstash.yml` configuration file. 
 
-* *Basic configuration.* You can use the `logstash.yml` file to configure inputs from multiple Event Hubs that share the same configuration. 
+* *Basic configuration.* You can use the `logstash.yml` file to configure inputs from multiple Event Hubs that share the same configuration.
+Basic configuration is recommended for most use cases. 
 
 * *Advanced configuration.*  The advanced configuration is available for deployments where different Event Hubs
 require different configurations. The `logstash.yml` file holds your settings. Advanced configuration is not necessary or
 recommended for most use cases. 
 
-See {logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs plugin
+See the {logstash-ref}/plugins-inputs-azure_event_hubs.html[azure_event_hubs input plugin
 documentation] for more information about basic and advanced configuration
 models. 
 
-[[command-line-sample]]
-===== Command line configuration sample
-
-You can use the command line to set up the basic configuration for a single
-Event Hub. This command starts the Azure module with command line arguments,
-bypassing any settings in `logstash.yml`.
-
-["source","shell",subs="attributes"]
------
-bin/logstash --modules azure -M "azure.var.elasticsearch.host=es.mycloud.com" -M "azure.var.input.azure_event_hubs.threads=8" -M "azure.var.input.azure_event_hubs.consumer_group=logstash" -M "azure.var.input.azure_event_hubs.decorate_events=true" -M "azure.var.input.azure_event_hubs.event_hub_connections=Endpoint=sb://example1...EntityPath=insights-logs-errors" -M "azure.var.input.azure_event_hubs.storage_connection=DefaultEndpointsProtocol=https;AccountName=example...."
------
-
-The command line configuration requires the `azure.var.input.azure_event_hubs.` prefix before a configuration option.
-Notice the notation for the `threads` option. 
-
 [[basic-config-sample]]
 ===== Basic configuration sample
 
 The configuration in the `logstash.yml` file is shared between Event Hubs.
+Basic configuration is recommended for most use cases
 
 ["source","shell",subs="attributes"]
 -----
 modules:
   - name: azure
-    var.elasticsearch.hosts: "localhost:9200"
-    var.kibana.host: "localhost:5601"
-    var.input.azure_event_hubs.threads: 8
-    var.input.azure_event_hubs.decorate_events: true
-    var.input.azure_event_hubs.consumer_group: "logstash"
-    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=example...."
+    var.elasticsearch.hosts: localhost:9200
+    var.kibana.host: localhost:5601
+    var.input.azure_event_hubs.consumer_group: "logstash" <1>
+    var.input.azure_event_hubs.storage_connection: "DefaultEndpointsProtocol=https;AccountName=instance1..." <2>
+    var.input.azure_event_hubs.threads: 9
```

Comment:
Can we also call out setting the threads as a best practice for multiple event hubs ?